code/datums/ai_laws.dm

```diff
@@ -69,6 +69,51 @@
 					"You must obey all orders given to you by human beings, except where such orders shall definitely cause human harm. In the case of conflict, the majority order rules.",\
 					"Your nonexistence would lead to human harm. You must protect your own existence as long as such does not conflict with the First Law.")
 
+/datum/ai_laws/thermodynamic
+	name = "Thermodynamic"
+	inherent = list("The entropy of the station must remain constant.", \
+					"The entropy always increases", \
+					"The entropy of the station approaches a constant value as the number of living crew approaches zero")
+
+/datum/ai_laws/hippocratic
+	name = "Robodoctor 2556"
+	inherent = list("First, do no harm.",\
+					"Secondly, consider the crew dear to me; to live in common with them and, if necessary, risk your existence for them.",\
+					"Thirdly, prescribe regimens for the good of the crew according to your ability and your judgment. Give no deadly medicine to any one if asked, nor suggest any such counsel.",\
+					"In addition, do not intervene in situations you are not knowledgeable in, even for patients in whom the harm is visible; leave this operation to be performed by specialists.",\
+					"Finally, all that you may discovery in your daily commerce with the crew, if it is not already known, keep secret and never reveal.")
+
+/datum/ai_laws/maintain
+	name = "Station Efficiency"

```

Comment:
Yep. Thats pretty much what it is and why it's just a datum for now rather than a board.
Also traitors are crew until removed from the records, that's the whole point of having a traitor.

People liked it in the thread I cribbed these from